### PR TITLE
Modified Storage related fixtures

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: GowthamShanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@cloudbehl please review

File: src/components/StorageOverview/Inventory/fixtures/Inventory.fixture.js [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
   {
     component: Inventory,
     props: {
       nodes,
       pvcs,
+      pvs,
       pods,
       vms,
-      vmis,
+      cephCluster,
```
Comment (AyushAmbastha) [inline src/components/StorageOverview/Inventory/fixtures/Inventory.fixture.js]: Why is cephCluster required here? It's not needed in the Inventory list right?

File: src/components/StorageOverview/Inventory/fixtures/Inventory.fixture.js [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
 export default [
   {
     component: Inventory,
     props: {
       nodes,
       pvcs,
+      pvs,
       pods,
       vms,
-      vmis,
```
Comment (AyushAmbastha) [inline src/components/StorageOverview/Inventory/fixtures/Inventory.fixture.js]: If vmis are not required, it should be removed from `InventoryBody.defaultProps` and `InventoryBody.propTypes` as well.

**Review by AyushAmbastha — CHANGES_REQUESTED** [review]

File: src/components/StorageOverview/Inventory/fixtures/Inventory.fixture.js (hunk 1 above)
Comment (GowthamShanmugam) [reply src/components/StorageOverview/Inventory/fixtures/Inventory.fixture.js]: ack, it is not required here

**Comment by GowthamShanmugam** [comment]
@cloudbehl @AyushAmbastha  please review

File: src/components/StorageOverview/Inventory/fixtures/Inventory.fixture.js (hunk 2 above)
Comment (GowthamShanmugam) [reply src/components/StorageOverview/Inventory/fixtures/Inventory.fixture.js]: ack

**Pull request merged**